app/src/main/java/org/jellyfin/androidtv/preferences/ui/UserPreferencesFragment.kt

```diff
@@ -0,0 +1,145 @@
+package org.jellyfin.androidtv.preferences.ui
+
+import android.app.AlertDialog
+import android.content.SharedPreferences
+import android.content.SharedPreferences.OnSharedPreferenceChangeListener
+import android.os.Bundle
+import androidx.preference.*
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.model.LogonCredentials
+import org.jellyfin.androidtv.util.DeviceUtils
+import org.jellyfin.androidtv.util.apiclient.AuthenticationHelper
+import java.io.IOException
+
+class UserPreferencesFragment : PreferenceFragmentCompat(), OnSharedPreferenceChangeListener {
```

Comment:
Should we be using `LeanbackPreferenceFragmentCompat`?

Reply:
LeanbackPreferenceFragmentCompat unfortunately doesn't work, when I tried it previously it crashed the application. So for now I didn't change the ui much. I might do that in the future though.

Reply:
I don’t like that this doesn’t match the app theme. Maybe we should extend the default theme you used to apply the Jellyfin colors as a temporary solution?

Reply:
I managed to get the leanback style preferences working. It now looks like this:

![image](https://user-images.githubusercontent.com/2305178/72679960-e80e5880-3ab4-11ea-9989-03ac3ac93ffe.png)